tracer/src/Datadog.Tracer.Native/tracer_tokens.cpp

```diff
@@ -144,6 +143,49 @@ const shared::WSTRING& TracerTokens::GetCallTargetReturnGenericType()
     return managed_profiler_calltarget_returntype_generics;
 }
 
+HRESULT TracerTokens::EnsureBaseCalltargetTokens()
+{
+    HRESULT hr = CallTargetTokens::EnsureBaseCalltargetTokens();
+    if (FAILED(hr))
+    {
+        return hr;
+    }
+    // *** Ensure Datadog.Trace.ClrProfiler.CallTarget.CallTargetBubbleUpException type ref, might not be available if tracer version is < 2.22
+    if (bubbleUpExceptionTypeRef == mdTypeRefNil)
+    {
+        const ModuleMetadata* module_metadata = GetMetadata();
+        module_metadata->metadata_emit->DefineTypeRefByName(profilerAssemblyRef,
+                                                            calltargetbubbleexception_tracer_type_name.c_str(),
+                                                            &bubbleUpExceptionTypeRef);
+    }
+    return hr;
+}
+
+int TracerTokens::GetAdditionalLocalsCount()
+{
+    // 1 for exception filter value
```

Comment:
Can you update the comment with what the second additional local is for?

Reply:
fixed in b9b2c8c95e0dc5c966184933b404f5d1ed7d06ba